### ISSUE TITLE
Library cannot be used in iOS v13 and older 

### ISSUE THREAD

**Issue by bowozer** [szimek/signature_pad#580]
It happened ever since using event emitter that merged by this PR https://github.com/szimek/signature_pad/pull/567

i setup a simple html script that opened from iPhone (iOS 13.6) emulator. The script pretty much the same with the fiddler on the README.md (https://jsfiddle.net/szimek/jq9cyzuc/), but i wrapped the SignaturePad object creation with try-catch to see the error.

Original Code:  
```js
var signaturePad = new SignaturePad(canvas, {
  backgroundColor: 'rgb(255, 255, 255)' // necessary for saving image as JPEG; can be removed is only saving as PNG or SVG
});
```
Modified to:
```js
var signaturePad;
try {
    signaturePad = new SignaturePad(canvas, {
        backgroundColor: 'rgb(255, 255, 255)' // necessary for saving image as JPEG; can be removed is only saving as PNG or SVG
    });
} catch (error) {
    alert(`oh no '${error.message}', it means current browser does not have EventTarget() constructor..`);
}
```

Here is the error:
![image](https://user-images.githubusercontent.com/11330467/145159080-80f7fd9d-5619-49fc-a066-cc11b8d7660f.png)

I think it is because the SignaturePad inherits from EventTarget and call EventTarget constructor. Hence the "super()" error.  
iOS only start support EventTarget constructor from iOS 14....  
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/eventTarget

**Comment by bowozer** [comment]
oh yeah i probably should say: user cannot stroke any lines in the canvas after that error. I guess it is pretty much breaking the pad.

I think it is important to make it runs on iOS 13 because many devices still works on them.

**Comment by bowozer** [comment]
i hope this PR https://github.com/szimek/signature_pad/pull/581 suffices to solve this issue. Just awaiting for approval

**Comment by welsh** [comment]
I noticed on Android 6.0.1 that also upgrading it stopped working, reverting back to `v3.0.0-beta.3` resolved the problem.

The error was about an illegal constructor typescript, I wonder if this is the same issue that you're running into.

**Comment by bowozer** [comment]
@welsh well luckily on Android OS, the browser engine actually decoupled with the OS itself. So you just need to upgrade the browser to the latest and it would have the newest browser engine, and thus it would work with the latest library. 

Unfortunately, it is different with iOS, though. No matter the browser in iOS, [it always uses webkit engine](https://en.wikipedia.org/wiki/Comparison_of_browser_engines_(HTML_support)#Explanation_of_the_tables) . The engine (webkit) is coupled with the iOS. That is why to upgrade browser engine, you would need to upgrade iOS as well.. It doesn't sit really well because it would make this library unusable in many devices that currently being used.

**Comment by big-silly** [comment]
For the current problem, my solution is to introduce an [EventTarget polyfill](https://github.com/ungap/event-target) in the head tag of html.

**Comment by UziTech** [comment]
fixed by #581
